tests/TestEntry.cpp

```diff
@@ -409,8 +420,86 @@ void TestEntry::testResolveNonIdPlaceholdersToUuid()
         newEntry.setGroup(root);
         newEntry.setNotes(newEntryNotesRaw);
 
-        const auto newEntryNotesResolved = 
+        const auto newEntryNotesResolved =
             newEntry.resolveMultiplePlaceholders(newEntry.notes());
         QCOMPARE(newEntryNotesResolved, QString(referencedEntry->uuid().toHex()));
     }
 }
+
+void TestEntry::testResolveClonedEntry()
+{
+    Database db;
+    Group* root = db.rootGroup();
+
+    Entry *original = new Entry();
```

Comment:
@cyphar we normally place pointers on the left, just like `Group* root` above.

Reply:
Fixed.